app/assets/stylesheets/mortgage_calculator/components/_stamp-duty-calc.scss

```diff
@@ -136,3 +138,59 @@ input.stamp-duty__input {
 .stamp-duty__have-you-tried {
   @include body(20, 24);
 }
+
+.stamp-duty__calculator-column {
+  display: inline;
+  float: left;
+  width: 50%;
+
+  margin-top: $baseline-unit*2;
+
+  border-right: 1px solid $color-grey-pale;
+  padding-right: $baseline-unit*4;
+  margin-bottom: $baseline-unit*4;
+}
+
+.stamp-duty__info-column {
+  display: inline;
+  float: left;
+  width: 50%;
+
+  padding-left: $baseline-unit*4;
+  margin-top: $baseline-unit*2;
+}
+
+.stamp-duty__results {
+  background: $color-grey-pale;
+  padding: $baseline-unit*3;
+}
+
+.stamp-duty__results--heading {
```

Comment:
This should be ``stamp-duty__results-heading`` as 'heading' is an element inside the ``stamp-duty__results`` block and not actually a modifier of ``stamp-duty__results``